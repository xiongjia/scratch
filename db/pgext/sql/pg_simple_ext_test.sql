CREATE EXTENSION pg_simple_ext;
SELECT my_simple_ext();

