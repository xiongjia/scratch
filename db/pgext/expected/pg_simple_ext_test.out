CREATE EXTENSION pg_simple_ext;
SELECT my_simple_ext();
 my_simple_ext 
---------------
 simple ext
(1 row)

